tests/auto_beam_formatting_tests.ts

```diff
@@ -623,7 +623,7 @@ function beamingWithSeveralGroupsOverflow3(options: TestOptions): void {
 
   f.draw();
 
-  beams.forEach((beam) => beam.setContext(f.getContext()).draw());
+  beams.forEach((beam) => beam.setContext(f.getContext()).drawWithStyle());
```

Comment:
One question: Does calling .draw() still work? Or do we have to go through .drawWithStyle() from now on?

Reply:
draw works without applying the styles